ibm-ups/monitor.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright © 2021 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "monitor.hpp"
+
+namespace phosphor::power::ibm_ups
+{
+
+/**
+ * D-Bus service/bus name for this application.
+ */
+constexpr auto serviceName = "xyz.openbmc_project.Power.IBMUPS";
```

Comment:
Usually when we have IBM specific service names they start with org.open_power.  Could you doublecheck it with Brad?

Reply:
since we will never upstream this application, I don't really have a preference.

Reply:
Good question.  I had discussed that potential service name with Brad last month, and he said it was OK.  I believe the only thing that will be IBM-specific about the application is the use of the USB cable.  I'm not sure if similar cables are used to connect UPS devices on other systems.

Reply:
It already has IBM in the name, so I was thinking that implied it was IBM specific.  anyway as Brad said since this will never be upstreamed I guess I don't care.

Reply:
Yeah, good point.  I'll look into changing it in a future PR.  I'd like to get this merged since PHYP is waiting on it.  I need to test that a name with a different domain is still seen properly by the mapper since it has a white list and black list.